lib/info.js

```diff
@@ -94,6 +96,7 @@ module.exports = function getInfo(link, options, callback) {
       });
       myrequest(url, options.requestOptions, function(err, body) {
         if (err) return callback(err);
+        relatedIds = getRelatedVideos(body);
```

Comment:
The body of this request is a querystring, no need to look for related videos here.